content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -136,7 +136,7 @@ HSV color
 lightness
 : Mix lightness from the input and output images, while taking color (chroma and hue) unaltered from the input image.
 
-chrominance
+chromaticity
 : Mix chrominance from the input and output images, while taking lightness and hue unaltered from the input image. This blend mode uses RGB ratios, divided by a Euclidean norm.
```

Comment:
presumably we also need to change the word chrominance in the definition.

Reply:
Yeah, maybe we need to ask the developer or the author to check it, but I don't know who.

Reply:
@aurelienpierre says chromaticity is "not wrong" so let's go with that. I'll merge and then copy-edit after.